views/index.ejs

```diff
@@ -61,7 +61,8 @@
       <%- include navbar %>
       <%- include overlay %>
       <%- include title %>
-      <div class="g mnone">
+      //SplitJS
```

Comment:
EJS is basically HTML with extra stuff, so you have to do comments with `<!--- comment --->`